### PR TITLE
Fixes to the Lease view screen

### PR DESCRIPTION
**Pull request by FuriousLlama**

### CODE REVIEW

File: source/frontend/src/features/leases/detail/LeasePages/details/DetailAdministration.tsx [hunk 1]
```diff
@@ -65,10 +67,7 @@ export const DetailAdministration: React.FunctionComponent<
           />
         </SectionField>
         <SectionField label="Effective date" labelWidth="3">
-          <Input
-            disabled={disabled}
-            field={withNameSpace(nameSpace, 'responsibilityEffectiveDate')}
-          />
+          {prettyFormatDate(responsabilityDate)}
```
Comment (devinleighsmith) [inline source/frontend/src/features/leases/detail/LeasePages/details/DetailAdministration.tsx:70]: nit: responsibilityDate

**Comment by devinleighsmith** [comment]
Not seeing this requirement satisfied: PID should have it's own space to be displayed, not following Legal description content

**Comment by devinleighsmith** [comment]
@FuriousLlama this doesn't seem fixed either unless I'm missing something: **Initiator data is not displayed even an option has been selected.**

**Comment by devinleighsmith** [comment]
Did this get fixed? **Responibility field displays "Headquarter" on the View Form if the user doesn't select any option in particular (Option left with default value "Select group responsible")**

**Comment by devinleighsmith** [comment]
@FuriousLlama this still seems to be a problem as well: **In case an address is not present, the "Address not available in PIMS" should be located next to the Address label**

**Review by devinleighsmith — CHANGES_REQUESTED** [review]
see comments, not all AC items addressed yet.

**Review by devinleighsmith — APPROVED** [review]
Based on discussion giving approval so that if you want to merge once you fix the actual bugs you can.

File: source/frontend/src/features/leases/detail/LeasePages/details/LeaseDetailsForm.tsx [hunk 2]
```diff
@@ -12,21 +12,19 @@ import styled from 'styled-components';
...
   return (
     <Formik initialValues={{ ...defaultLease, ...lease }} enableReinitialize={true} onSubmit={noop}>
       <StyledDetails>
         <DetailTermInformation />
-        <PropertiesInformation disabled={!isEditing} />
-        <DetailAdministration disabled={!isEditing} />
-        <DetailDocumentation disabled={!isEditing} />
+        <PropertiesInformation disabled={true} />
+        <DetailAdministration disabled={true} />
+        <DetailDocumentation disabled={true} />
```
Comment (FuriousLlama) [inline source/frontend/src/features/leases/detail/LeasePages/details/LeaseDetailsForm.tsx:27]: This needs to be refactored given the underlying components are only used for view

**Pull request merged**